### PR TITLE
feat: i add animation to show the modal with transition and fix the icon of close in modal component

### PR DESCRIPTION
**Pull request by Gabojcb**
Hi, development Team.

i upload change about modal component, i add efect animation to modal with transition, besides also ajustment height and width of button icon close in modal.

By Gabojcb.

### CODE REVIEW

**Review by Gabojcb — COMMENTED** [review]
I I reviewed and tested the commit

**Comment by Gabojcb** [comment]
I reviewed and tested the commit

**Review by Gabojcb — COMMENTED** [review]
I reviewed and tested the commit

File: project/modules/modal/ts/modal.tsx
Comment (Gabojcb) [inline project/modules/modal/ts/modal.tsx:1]: I reviewed

File: project/modules/modal/ts/children.tsx
Comment (Gabojcb) [inline project/modules/modal/ts/children.tsx:1]: I reviewed also

File: project/modules/modal/scss/close-icon.scss
Comment (Gabojcb) [inline project/modules/modal/scss/close-icon.scss:1]: I reviewed also

File: project/modules/modal/scss/modal.scss
Comment (Gabojcb) [inline project/modules/modal/scss/modal.scss:1]: I reviewed also for accept changes and verify

**Review by Gabojcb — COMMENTED** [review]
E:\workspace\baleares\public\pragmate-ui\project\modules\modal\ts\children.tsx

**Review by jircdev — COMMENTED** [review]
Perfect!

**Pull request closed without merging**